llvm/docs/Contributing.rst

```diff
@@ -73,15 +73,33 @@ recent commit:
 
   % git clang-format HEAD~1
 
-Note that this modifies the files, but doesn't commit them -- you'll likely want
-to run
+.. note::
+  For some patches, formatting them may add changes that obscure the intent of
+  the patch. For example, adding to an enum that was not previously formatted
+  may result in the entire enum being reformatted. This happens because not all
+  of the LLVM Project conforms to clang-format at this time.
```

Comment:
[nit] `clang-format` is just a tool. Conform to `clang-format style`? `clang-format set-up file`? `clang-format setting`?
```suggestion
  of the LLVM Project conforms to clang-format set-up at this time.
```

Reply:
Done